src/renderer/components/wallet/assets/AssetsTableCollapsable.tsx

```diff
@@ -275,7 +275,10 @@ export const AssetsTableCollapsable: React.FC<Props> = (props): JSX.Element => {
   useEffect(() => {
     // don't change openPanelKeys if user has already changed panel state
     if (!collapseChangedByUser) {
-      const keys = chainBalances.map((_, i) => i.toString())
+      const keys: string[] = []
+      chainBalances.forEach((chainBalance, index) => {
+        if (RD.isSuccess(chainBalance.balances) && chainBalance.balances.value.length) keys.push(index.toString())
+      })
```

Comment:
Don't access the "inside" value by using `chainBalance.balances.value`. You can do it like the following (check the comments, you can remove those later in code)

```
      const keys = FP.pipe(
        chainBalances,
        // get `balances: WalletBalancesRD` from `chainBalances`
        A.map(({ balances }) => balances),
        // Filter `success` balances and get its value (WalletBalance) from it
        A.filterMap(RD.toOption),
        // Filter empty list of balances and get a `key` from `i` (index)
        A.filterMapWithIndex((i, balances) => (balances.length > 0 ? O.some(i.toString()) : O.none))
      )
```